src/systempolicy/systemPolicy.go

```diff
@@ -1146,6 +1186,8 @@ func GenFileSetForAllPodsInCluster(clusterName string, pods []types.Pod, settype
 			log.Error().Msgf("failure add/updt db entry for wpfs=%+v err=%s", wpfs, err.Error())
 		}
 	}
+
+	return true
```

Comment:
Hey, the return true should not be done blindly. Only if the DB is changed then the return should be true.